### PR TITLE
Adding support for ZergNet to AMP

### PR DESCRIPTION
**Pull request by zergnetjake**
I'm not trying to be irritating.  I removed the old pull request and created this new one in hopes of making things easier.

### CODE REVIEW

**Comment by zergnetjake** [comment]
I signed it!

**Comment by jridgewell** [comment]
@lannka: You're taking the Ads PRs now, right?

**Comment by lannka** [comment]
@jridgewell yes, you can assign to me. thanks for the triage.

File: ads/zergnet.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# ZergNet
+
+## Example
+
+### Basic
+
+```html
+<amp-embed
+	width="780" height="100"
```
Comment (lannka) [inline ads/zergnet.md]: nit: let's indent 4 spaces :-)

**Comment by lannka** [comment]
One minor comment. Otherwise LGTM.

Not sure what happened to CLA though. Could you retry?

**Comment by zergnetjake** [comment]
My corporate email is jake@zergnet.com, I created my github account using an alias to that account, thejake@zergnet.com.  There does not appear to be a way to sign the CLA as thejake@zergnet.com, as our email is Google-hosted and it sets my identity to jake@zergnet.com automatically.

I can delete this pull request and submit again, signing all commits as jake@zergnet.com, but that is not a real github account, unless I also create that.

I will do whatever I need to in order to make things easy on you, please advise.

File: ads/zergnet.md (hunk 1 above)
Comment (zergnetjake) [inline ads/zergnet.md]: Changed as requested.

**Comment by lannka** [comment]
@cramforce can you advice? you have more experience on this issue :-)

**Comment by cramforce** [comment]
Checked CLA. Looks good. Merging.

**Pull request merged**